driver/razermousemat_driver.c

```diff
@@ -620,11 +650,13 @@ static int razer_mousemat_probe(struct hid_device *hdev, const struct hid_device
     if(intf->cur_altsetting->desc.bInterfaceProtocol == USB_INTERFACE_PROTOCOL_MOUSE) {
 
         switch(usb_dev->descriptor.idProduct) {
+        case USB_DEVICE_ID_RAZER_FIREFLY_HYPERFLUX:
         case USB_DEVICE_ID_RAZER_GOLIATHUS_CHROMA:
         case USB_DEVICE_ID_RAZER_GOLIATHUS_CHROMA_EXTENDED:
             CREATE_DEVICE_FILE(&hdev->dev, &dev_attr_version);
             CREATE_DEVICE_FILE(&hdev->dev, &dev_attr_matrix_custom_frame);
             CREATE_DEVICE_FILE(&hdev->dev, &dev_attr_matrix_reactive_trigger);
+            CREATE_DEVICE_FILE(&hdev->dev, &dev_attr_matrix_effect_wave);
```

Comment:
Hmm this adds the wave effect for the Goliathus models as well, but as you said, Wave doesn't work on this so I'll remove it for now